mobile/native/src/db/models.rs

```diff
@@ -221,6 +222,19 @@ impl Order {
             .load(conn)
     }
 
+    /// Gets any force order in the database. A force order is defined by any order with an reason
+    /// not equal to `Manual`
```

Comment:
I'm not buying this concept of a _forced_ order yet... The naming does not align in my brain 🙈

Reply:
renamed to async order